orchagent/routeorch.cpp

```diff
@@ -1085,18 +1321,21 @@ bool RouteOrch::removeRoute(sai_object_id_t vrf_id, const IpPrefix &ipPrefix)
         return true;
     }
 
+    sai_route_entry_t route_entry;
+    route_entry.vr_id = vrf_id;
+    route_entry.switch_id = gSwitchId;
+    copy(route_entry.destination, ipPrefix);
+
     auto it_route = it_route_table->second.find(ipPrefix);
-    if (it_route == it_route_table->second.end())
+    size_t creating = gRouteBulker.creating_entries_count(route_entry);
+    if (it_route == it_route_table->second.end() && creating == 0)
```

Comment:
So I thought you would want to remove this entry from Bulker if there is a remove Route and `creating `is `True`

Reply:
> if there is a remove Route and creating is True

In this case, this if-block will be skipped, and it may be removed from Bulker if it happen to be there. However upstream merging in consumer.m_toSync will make this SET->DEL impossible to happen.

Did you see a bug?

---
In reply to: [410516175](https://github.com/Azure/sonic-swss/pull/1238#discussion_r410516175) [](ancestors = 410516175)